### ISSUE TITLE
Show image's rendered size when zooming in/out

### ISSUE THREAD

**Issue by kevinfarrugia** [GoogleChromeLabs/squoosh#1305]
**Is your feature request related to a problem? Please describe.**
When compressing an image, I usually zoom out to get the image to be of a similar size to how it would be rendered on the page. This way I know that the compression levels I am choosing are suitable for how this image will be displayed on the page.

**Describe the solution you'd like**
It would be extremely helpful if in addition to the zoom percentage, it also shows the rendered image size (in pixels) as you zoom in/out. I think this could be achieved by multiplying the image's original size with the `--scale`.

**Does other service/app have this feature?**
N/A

**Comment by jakearchibald** [comment]
I like this idea!

**Comment by macx** [comment]
Yes, I miss with feature every single time I'm using squoosh. This should be default. The preview should also reflect the new dimensions when resizing.
